packages/deployer/src/deploySolutionItems.ts

```diff
@@ -694,6 +710,33 @@ export function _createItemFromTemplateWhenReady(
   return itemDef;
 }
 
+// export function _createItemFromTemplateWhenReadyNew(
```

Comment:
Cut?

Reply:
yeah - sorry - I should have nixed this stuff out